common/app/views/fragments/bashers/ThisIsTheNHS.scala.html

```diff
@@ -0,0 +1,41 @@
+@()(implicit request: RequestHeader)
+
+<div class="gs-container">
+    <div class="this-is-the-nhs">
+        <h1 class="title title--first">this is</h1>
+        <h1 class="title title--second">the NHS</h1>
```

Comment:
I know guys :disappointed:. that's why I deleted the comment because @sammorrisdesign dumb.